packages/marshal/test/test-passStyleOf.js

```diff
@@ -129,7 +128,5 @@ test('passStyleOf testing remotables', t => {
   const farObj6 = harden({
     __proto__: farObjProto6,
```

Comment:
I don't have strong feeling about this, so am happy to switch if you do.

The reasons I like it better:
   * It is syntax, and so its meaning is invariant of any bindings in the local lexical scope.
   * The properties of the object, which I admit are absent here, can appear using normal object-literal property syntax, rather than descriptor syntax, which is more pleasant when that it all you need.

I'll leave this unresolved until I hear back.

Reply:
Took your suggested helper. Definite improvement. Thanks. Done.